go/bundle/version/version.go

```diff
@@ -0,0 +1,65 @@
+package version
+
+import (
+	"bytes"
+	"errors"
+	"io"
+)
+
+type Version string
+
+const (
+	Unversioned Version = "unversioned"
+	VersionB1 Version = "b1"
+)
+
+// The CBOR encoding of the 4-item array initial byte and 8-byte bytestring initial byte, followed by 🌐📦 in UTF-8.
+var HeaderMagicBytesUnversioned = []byte{0x84, 0x48, 0xf0, 0x9f, 0x8c, 0x90, 0xf0, 0x9f, 0x93, 0xa6}
+// The CBOR encoding of the 6-item array initial byte and 8-byte bytestring initial byte, followed by 🌐📦 in UTF-8.
+var HeaderMagicBytes = []byte{0x86, 0x48, 0xf0, 0x9f, 0x8c, 0x90, 0xf0, 0x9f, 0x93, 0xa6}
+
+// The CBOR encoding of a 4-byte byte string holding an ASCII "b1" followed by two 0 bytes
```

Comment:
@jyasskin What implementation-specific version string should we use?

Reply:
I’d take “b1\0\0” for now, roughly matching signed exchanges.

Reply:
OK, let's use "b1\0\0" for now.